backend/benefit/common/permissions.py

```diff
@@ -53,3 +53,22 @@ def has_permission(self, request, view):
                 # proceed without a company.
                 raise PermissionDenied(_("Company information is not available"))
             return not TermsOfServiceApproval.terms_approval_needed(user, company)
+
+
+class SafeListPermission(permissions.BasePermission):
```

Comment:
So we are going to have IP restrictions here. I think it's better as we have better control over them here :D Should we then ditch the platta-made IP restrictions and implement everything here?

Reply:
For the time being, yeah I suppose let's enforce the IP here, we can refactor this away later if we get the platta IP restrictions to work as intended at some point.